dataland-frontend/tests/e2e/specs/eu-taxonomy-financials/FileUploadAndLinking.ts

```diff
@@ -45,21 +46,25 @@ describeIf(
         () => {
           uploadDocuments.selectFile(TEST_PDF_FILE_NAME);
```

Comment:
The name of this test case is harmless, but it actually spends a huge amount of its time filling out the upload form. So much so, that it manages to crash my local Cypress instance. I do not think this is required for checking if document linking is working as expected... (Nevertheless, refactoring this could be out of scope of this ticket, so I would be fine with creating a follow up or discussing it)

Reply:
Maybe a separate refactoring story for this is needed.

Reply:
Would argue in favour of a new story for this

Reply:
Can you create one?

Reply:
Created:
https://jira.d-fine.dev/browse/DALA-2397